### DISCUSSION TITLE
Pores and irregularities on the skin

### DISCUSSION THREAD

**Discussion by Treawolfe** [lllyasviel/Fooocus#1947] [Q&A]
I hope there isn't already a similar topic, but I'm wondering how I can get pores and irregularities, perhaps even pimples, to appear on my skin. I think the images that are generated look very unnatural if you pay attention to details, a little more natural look would help, the problem is that the AI doesn't implement my input. I can put it the way I want, I always get absolutely smooth (photoshop exaggerated) skin

"Young Norwegian girl with long blonde hair takes selfie while walking in winter. Skin with pores, white hat, gray winter coat,"
This is an example that I tried, but I still only get pictures of faces that are comparable to a top model and that have been extremely photoshopped. I've tried many other options but somehow it doesn't seem to work properly

**Comment by poor7** [comment]
Can you provide an example of what exactly you need in the photo?
I have no problems with irregularities on the skin:
![2024-01-17_04-38-22_8232](https://github.com/lllyasviel/Fooocus/assets/154618384/92809c81-5f15-4775-a88c-6c967fc6559c)

**Reply by Treawolfe** [reply]
It's possible that I'm using the wrong prompts, but I don't know which ones would work. I write things like "visible skin pores, acne, pimples, blemishes" etc... but I always get smooth, far too perfect faces, a picture similar to the one you showed has never been generated for me.
Which prompts do you use and how do you mention them? Do you write complete sentences or just bullet points?

**Comment by psydox** [comment]
No issues on my side as well.  Using Fooocus v 2.1.862.


Prompt: A young Norwegian girl with long blonde hair takes a selfie while walking in winter. Skin with pores, white hat, gray winter coat, ((mild red pimples on the face))

Styles: None

![2024-01-18_02-19-08_8433](https://github.com/lllyasviel/Fooocus/assets/5681500/3a152f67-d042-467f-8f1f-0579c17970c5)



Prompt: A young Norwegian girl with long blonde hair takes a selfie while walking in winter. Skin with pores, white hat, gray winter coat, ((mild red pimples on the face))

Styles: Fooocus V2, Fooocus Enhance, Fooocus Sharp

![2024-01-18_02-20-50_1785](https://github.com/lllyasviel/Fooocus/assets/5681500/18bb7a9b-f1a2-4586-b201-3bb910bc55d8)

**Reply by Treawolfe** [reply]
I've now tried it exactly with the one you used but I get the same result again. Is it possible that such cosmetic errors don't work with faceswap?

(I would like to attach a picture of my result but somehow that doesn't work right now, in any case it's a face completely without blemishes, as if everything had been photoshopped away)
The settings I used this time: Fooocus V2 and faceswap.
Then I tried to generate something without faceswap, it works. Am I right in my assumption that it doesn't work with faceswap or can it work here too if you set some parameters correctly, the AI doesn't seem to listen to the prompts at this moment.
